project/api/models.py

```diff
@@ -43,6 +43,12 @@ class TerritorialEntity(models.Model):
 
     history = HistoricalRecords()
 
+    def get_children(self):
```

Comment:
Is there a way to rename `relations` and `political_relations` to `children` and `parents`? I tried that, but it seems that we need to change the field names in PoliticalRelation too.

Reply:
I'll look into it. Originally I wanted to avoid naming them as such, thinking it could be ambiguous to have similar names in the PR model.